pipeline/trusted-resources/pkg/trustedtask/trustedpipeline.go

```diff
@@ -0,0 +1,66 @@
+package trustedtask
+
+import (
+	"context"
+
+	"github.com/tektoncd/pipeline/pkg/apis/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"knative.dev/pkg/apis"
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+	"knative.dev/pkg/logging"
+)
+
+type TrustedPipeline struct {
+	v1beta1.Pipeline
+}
+
+func (p *TrustedPipeline) Validate(ctx context.Context) (errs *apis.FieldError) {
+	k8sClient := kubeclient.Get(ctx)
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return apis.ErrGeneric(err.Error())
+	}
+	tektonClient, err := versioned.NewForConfig(config)
+	if err != nil {
+		return apis.ErrGeneric(err.Error())
+	}
```

Comment:
If we're using injection clients for k8s, should we do the same for Tekton? https://pkg.go.dev/github.com/tektoncd/pipeline@v0.36.0/pkg/client/injection/client

Reply:
didn't realize this diff. thanks! 
let me update with `client.Get()`.